Packs/CommonPlaybooks/pack_metadata.json

```diff
@@ -1,292 +1,292 @@
-        },
-        "AlienVault_OTX": {
-            "mandatory": false,
-            "display_name": "AlienVault OTX"
-        },
-        "ThreatGrid": {
-            "mandatory": false,
-            "display_name": "Cisco Threat Grid"
-        },
-        "ThreatConnect": {
-            "mandatory": false,
-            "display_name": "ThreatConnect"
-        },
-        "Zscaler": {
-            "mandatory": false,
-            "display_name": "Zscaler"
-        },
-        "Maltiverse": {
-            "mandatory": false,
-            "display_name": "Maltiverse"
-        },
-        "EclecticIQ": {
-            "mandatory": false,
-            "display_name": "EclecticIQ Platform"
-        },
-        "Traps": {
-            "mandatory": false,
-            "display_name": "Palo Alto Networks Traps"
-        },
-        "PAN-OS": {
-            "mandatory": false,
-            "display_name": "PAN-OS"
-        },
-        "TruSTAR": {
-            "mandatory": false,
-            "display_name": "TruSTAR"
-        },
-        "epo": {
-            "mandatory": false,
-            "display_name": "McAfee ePO"
-        },
-        "Recorded_Future": {
-            "mandatory": false,
-            "display_name": "Recorded Future"
-        },
-        "EWSMailSender": {
-            "mandatory": false,
-            "display_name": "EWS Mail Sender"
-        },
-        "PolySwarm": {
-            "mandatory": false,
-            "display_name": "PolySwarm"
-        },
-        "Cisco-umbrella": {
-            "mandatory": false,
-            "display_name": "Cisco Umbrella Investigate"
-        },
-        "ImageOCR": {
-            "mandatory": false,
-            "display_name": "Image OCR"
-        },
-        "EWS": {
-            "mandatory": false,
-            "display_name": "EWS"
-        },
-        "XForceExchange": {
-            "mandatory": false,
-            "display_name": "IBM X-Force Exchange"
-        },
-        "URLHaus": {
-            "mandatory": false,
-            "display_name": "URLhaus"
-        },
-        "MISP": {
-            "mandatory": false,
-            "display_name": "MISP"
-        },
-        "Cybereason": {
-            "mandatory": false,
-            "display_name": "Cybereason"
-        },
-        "VMRay": {
-            "mandatory": false,
-            "display_name": "VMRay"
-        },
-        "HybridAnalysis": {
-            "mandatory": false,
-            "display_name": "Hybrid Analysis"
-        }
-    }
+{
+    "name": "Common Playbooks",
+    "description": "Frequently used playbooks pack.",
+    "support": "xsoar",
+    "currentVersion": "1.5.1",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-05-14T08:33:25Z",
+    "categories": [],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "dependencies": {
+        "VirusTotal": {
+            "mandatory": false,
```

Comment:
you'll need to override this by recalculating dependencies and setting fortinet to mandatory:False, like the rest.

Reply:
Done.